python/az/aro/azext_aro/tests/latest/test_aro_helpers.py

```diff
@@ -0,0 +1,24 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
```

Comment:
unit tests are failing because you are not using the same header that all the other files in this repo use